README.md

```diff
@@ -148,14 +149,6 @@ Feel free to check ......
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## ⭐️ Show your support <a name="support"></a>
-
-> Write a message to encourage readers to support your project
-
-If you like this project...
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
 > Give credit to everyone who inspired your codebase.
```

Comment:
- [ ] [OPTIONAL] you shouldn't leave the placeholders please make sure to fill it out with your information